src/Detector/Constants.cs

```diff
@@ -9,5 +9,6 @@ internal static class Constants
     {
         public const string FunctionApplications = "functions";
         public const string StaticSiteApplications = "static-sites";
+        public const string RelativeRootDirectory = "./";
```

Comment:
Does it need to be like this? can it be like "src/webapp" instead?

Reply:
Actually take a look at how `System.IO.Path.GetRelativePath` behaves here...I think it would return `.`

Reply:
Noticed for .NET standard 2.0, it doesn't have get relative path method. For the relative path, my understanding is, if a root repo path is `C:/apps` and absolute app directory is `C:/apps/src/webapp`, then it will output `./src/webapp`. And windows and linux expression are different, windows is like `.\src\webapps`, so we'll just nomalize it to be `./src/webapp`.